app/views/translation/table.js

```diff
@@ -68,7 +68,7 @@
             var translation = $scope.translation;
 
             var url = baseUrl+'translation-api/database-table/'+encodeURIComponent(translation.name);
-            var filesForTranslation = _.map(_.filter($scope.translation.rows, {translate:true}), '_id');
+            var filesForTranslation = _.map(_.filter($scope.articlesToDownload, {translate:true}), '_id');
```

Comment:
no need for filter, everything in acrticlesToDownload should be downloaded, remove filter all together